packages/pytest-simcore/src/pytest_simcore/redis_service.py

```diff
@@ -19,32 +19,46 @@
 log = logging.getLogger(__name__)
 
 
-@pytest.fixture(scope="function")
-async def redis_config(docker_stack: Dict, testing_environ_vars: Dict) -> RedisConfig:
+@pytest.fixture
+async def redis_settings(
+    docker_stack: Dict,  # stack is up
+    testing_environ_vars: Dict,
+) -> RedisSettings:
+    """Returns the settings of a redis service that is up and responsive"""
+
     prefix = testing_environ_vars["SWARM_STACK_NAME"]
     assert f"{prefix}_redis" in docker_stack["services"]
 
-    # test runner is running on the host computer
-    config = RedisConfig(
-        host="127.0.0.1",
-        port=get_service_published_port(
-            "simcore_redis", testing_environ_vars["REDIS_PORT"]
-        ),
+    port = get_service_published_port(
+        "simcore_redis", testing_environ_vars["REDIS_PORT"]
     )
-    await wait_till_redis_responsive(str(config.dsn))
-    return config
+    # test runner is running on the host computer
+    settings = RedisSettings(REDIS_HOST="127.0.0.1", REDIS_PORT=int(port))
```

Comment:
should we not set the host ip instead of 127.0.0.1? same reason as above.

Reply:
done